uat/testing-features/src/main/resources/greengrass/features/log-manager-1.feature

```diff
@@ -9,6 +9,18 @@ Feature: Greengrass V2 LogManager
         And 5 temporary rotated log files for component aws.greengrass.Nucleus have been created
         And 5 temporary rotated log files for component UserComponentA have been created
 
+    @local
+    Scenario: LogManager-1-T0: install the local custom component
+        Given I create a Greengrass deployment with components
+            | aws.greengrass.Cli        | LATEST |
+        And I deploy the Greengrass deployment configuration
+        Then the Greengrass deployment is COMPLETED on the device after 2 minutes
+        When I install the component logGenerator from local store with configuration
```

Comment:
I don't want to repeat the same problem that we have with our internal tests.

I want this to use MERGE/RESET format directly.


```
"""json
{
"MERGE": {"key": "value", "FileSize": "10MB"}
}
"""
```